gradle/libs.versions.toml

```diff
@@ -36,12 +26,26 @@ scala-library = "org.scala-lang:scala-library:2.13.8"
 slf4j-lambda = "kr.pe.kwonnam.slf4j-lambda:slf4j-lambda-core:0.1"
 slf4j-simple = "org.slf4j:slf4j-simple:1.7.36"
 vavr = "io.vavr:vavr:0.10.4"
+
+# Plugins need to appear here in two notations for the purpose of being used in precompiled scripts as dependencies
+# [https://docs.gradle.org/current/samples/sample_convention_plugins.html#things_to_note]
+
 # Plugins
-pluginPackages-aspectj-postCompileWeaving = { module = "io.freefair.gradle:aspectj-plugin", version.ref = "plugins-aspectj-postCompileWeaving" }
-pluginPackages-checkerFramework = { module = "org.checkerframework:checkerframework-gradle-plugin", version.ref = "plugins-checkerFramework" }
-pluginPackages-jetbrains-intellij = { module = "org.jetbrains.intellij.plugins:gradle-intellij-plugin", version.ref = "plugins-jetbrains-intellij" }
-pluginPackages-jetbrains-kotlin = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "plugins-jetbrains-kotlin" }
-pluginPackages-spotless = { module = "com.diffplug.spotless:spotless-plugin-gradle", version.ref = "plugins-spotless" }
+# @pin because 6.5.x is incompatible with kotlin plugin 1.4.x,
+#      see https://github.com/freefair/gradle-plugins/issues/554#issuecomment-1171434185 for more details
+pluginPackages-aspectj-postCompileWeaving = "io.freefair.gradle:aspectj-plugin:6.4.3.1"
```

Comment:
Try if it's possible to introduce some TOML subsection like `[pluginPackages]` or `[libraries.pluginPackages]` instead of prefixing each entry with `pluginPackages-`

Reply:
I think it's not, haven't seen that in any docs, and the compiler throws this when trying to introduce [pluginPackages]:
**Problem: In version catalog libs, unknown top level elements [pluginPackages]**